eng/common/scripts/Invoke-EmitterNpmInstall.ps1

```diff
@@ -0,0 +1,82 @@
+param(
```

Comment:
Why put this into its own script? I only see it used once.

Reply:
It's common to TypeSpec-Project-Generate and Cadl-Project-Generate

Reply:
Also, we want to be able to do emitter npm install once, then call `TypeSpec-Project-Generate.ps1` multiple times concurrently with the -SkipNpmInstall switch.   I can remove this optimization from the current PR as it's not a blocker.

Reply:
Script removed pending more conversation on tsp-client